src/app/sidebar/sidebar.component.scss

```diff
@@ -1,12 +1,15 @@
-h5{
+h5 {
   font-size: 18px;
 }
+
 .cursor-pointer {
   cursor: pointer;
 }
-li i{
-  font-size:30px;
+
+li i {
```

Comment:
it will be applied to all `i` tag which can inside the `li` tag. 
 do we have any other place this scenario ??